### PR TITLE
[Added]: Home Page

### PR DESCRIPTION
**Pull request by munjalSaksham71**
Description: Added Home Page and Links to the Product Page.

### CODE REVIEW

File: src/components/Categories/Categories.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { Link } from 'react-router-dom';
+import { useFilters } from '../../context/filter-context';
+import './Categories.css'
+const Categories = (props) => {
+    const {category} = props;
```
Comment (JantuDeb) [inline src/components/Categories/Categories.jsx]: You can destructure it in function argument.
```
Categories ({category})=>{}
```

File: src/components/Categories/Categories.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    const { productListDispatch } = useFilters()
+    return (
+        <div className="category">   
+            {category.map((c) => (
```
Comment (JantuDeb) [inline src/components/Categories/Categories.jsx]: I would suggest you replace `category` with `categories` and `c` with `category`

File: src/screens/HomeScreen.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import Categories from '../components/Categories/Categories';
+import Navbar from '../components/Nav/Navbar';
+
+const HomeScreen = () => {
+    const [category, setCategory] = useState([]);
+    const [error, setError] = useState("");
```
Comment (JantuDeb) [inline src/screens/HomeScreen.jsx:8]: This is really good that you added a state for error. You can take it to next lavel by using a loading state 🤠

**Review by JantuDeb — APPROVED** [review]
Overall the codes are well written

File: src/screens/HomeScreen.jsx (hunk 3 above)
Comment (munjalSaksham71) [reply src/screens/HomeScreen.jsx:8]: Ok, Ill do it.

File: src/components/Categories/Categories.jsx (hunk 2 above)
Comment (munjalSaksham71) [reply src/components/Categories/Categories.jsx]: Thanks for suggestion

File: src/backend/db/categories.js [hunk 4]
```diff
@@ -8,20 +8,30 @@ import { v4 as uuid } from "uuid";
...
+      "Chelsea boots are close-fitting, ankle-high boots with an elastic side panel. They often have a loop or tab of fabric on the back of the boot, enabling the boot to be pulled on. The boot dates back to the Victorian era, when it was worn by both men and women.",
   },
   {
     _id: uuid(),
-    categoryName: "non-fiction",
+    categoryName: "Casuals",
+    image: './Assets/tan_casual.jpeg',
     description:
-      "Non-fiction is writing that gives information or describes real events, rather than telling a story.",
+      "Casual shoes are often made in different colors especially in white, brown & black from materials like leather, suede, rubber, and other such materials that not only make them breathable but also lightweight.",
```
Comment (Manju-sree) [inline src/backend/db/categories.js:21]: good to see clear desriptions

File: src/screens/HomeScreen.jsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+import { useEffect, useState } from 'react';
+import Categories from '../components/Categories/Categories';
+import Navbar from '../components/Nav/Navbar';
+
+const HomeScreen = () => {
+    const [categories, setCategories] = useState([]);
+    const [error, setError] = useState("");
+    const [loading, setLoading] = useState(true);
+    useEffect(() => {
+        (async() => {
```
Comment (Manju-sree) [inline src/screens/HomeScreen.jsx:11]: nice your are using iffi and async func with try and catch blocks

**Review by Manju-sree — APPROVED** [review]
good to see that you have maintained clean code structure for each components.

**Pull request merged**